### PR TITLE
Use published version of the Java support libs

### PR DESCRIPTION
**Pull request by BoD**

### CODE REVIEW

File: tests/java-client/src/test/java/test/ApolloWsTest.java [hunk 1]
```diff
@@ -5,11 +5,11 @@
...
-import com.apollographql.apollo3.runtime.java.ApolloCallback;
-import com.apollographql.apollo3.runtime.java.ApolloClient;
-import com.apollographql.apollo3.runtime.java.ApolloDisposable;
-import com.apollographql.apollo3.runtime.java.network.ws.protocol.ApolloWsProtocol;
-import com.apollographql.apollo3.rx3.java.Rx3Apollo;
+import com.apollographql.java.client.ApolloCallback;
+import com.apollographql.java.client.ApolloClient;
+import com.apollographql.java.client.ApolloDisposable;
+import com.apollographql.java.client.network.ws.protocol.ApolloWsProtocol;
+import com.apollographql.java.rx3.Rx3Apollo;
```
Comment (martinbonnin) [inline tests/java-client/src/test/java/test/ApolloWsTest.java:12]: Move to `apollo-kotlin-java-support`?

File: scripts/rxjava3.main.kts [hunk 2]
```diff
@@ -1,61 +0,0 @@
-#!/usr/bin/env kotlin
-import java.io.File
```
Comment (martinbonnin) [inline scripts/rxjava3.main.kts:2]: 🧹

File: libraries/apollo-api-java/README.md [hunk 3]
```diff
@@ -1,3 +1,3 @@
 # Module apollo-api-java
  
-apollo-api-java contains adapters for Java Optional values. 
+This module has been moved to the [Java Support](https://github.com/apollographql/apollo-kotlin-java-support) repository and published under `com.apollographql.apollo:java-client`. Please update your dependencies.
```
Comment (martinbonnin) [inline libraries/apollo-api-java/README.md:3]:
```suggestion
This module has been moved to the [Java Support](https://github.com/apollographql/apollo-kotlin-java-support) repository and published under `com.apollographql.java:client`. Please update your dependencies.
```

Comment (BoD) [reply libraries/apollo-api-java/README.md:3]: Woops good catch! Totally right.

**Review by martinbonnin — APPROVED** [review]

**Pull request merged**